README.md

```diff
@@ -65,3 +66,8 @@
 <!-- prettier-ignore-end -->
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
+
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->

```

Comment:
Please do not change these lines - @viktoriussuwandi . It seems duplicated, could you please check?

Reply:
sorry @tungbq, the duplicate lines removed

Reply:
Thanks @viktoriussuwandi